marathon-runtime-requirements.txt

```diff
@@ -1,4 +1,8 @@
 -e git+https://github.com/f5devcentral/f5-cccl.git@03e22c4779ceb88f529337ade3ca31ddcd57e4c8#egg=f5-cccl
+cryptography==1.3.2
+idna==2.2
+pyasn1==0.2.2
```

Comment:
Do we have approvals for these packages? If not, can you please submit a FOSS request for it?

Reply:
They are already present in the .f5license file